server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/PullRequest.java

```diff
@@ -78,4 +78,12 @@ public void addComment(IssueComment comment) {
     public void addReview(PullRequestReview review) {
         reviews.add(review);
     }
+
+    public Integer getAdditions() {
```

Comment:
Isn't this already generated by lombok?

Reply:
At least my compiler required an explicit definition for it to work. I assume it has to do with the fact that the variables are of the primitive type `int`, while the methods require an `Integer` as return value.

Feel free to try if it works without them for you though.